### PR TITLE
refactor(app): update set wifi ssid screen

### PR DESCRIPTION
**Pull request by koji**
# Overview
[design]
https://www.figma.com/file/AoTLAYuWawlaWItB1umOjr/Release%3A-Opentrons-Flex-Touchscreen-Designs?type=design&node-id=7218-91809&t=MfY2SBavbCe7JMQp-0

update set wifi ssid screen

# Test Plan
1. Go to Network setup menu
2. Tap `Wi-Fi`
3. Tap `Join other network`

# Changelog
- update SetWifiSsid component

# Review requests

# Risk assessment
low

### CODE REVIEW

File: app/src/organisms/OnDeviceDisplay/SetupNetwork/SetWifiSsid.tsx [hunk 1]
```diff
@@ -3,34 +3,47 @@ import { useTranslation } from 'react-i18next'
...
 import type { NetworkChangeState } from '../../Devices/RobotSettings/ConnectNetwork/types'
 
 const SSID_INPUT_FIELD_STYLE = css`
-  padding-top: ${SPACING.spacing24};
-  padding-bottom: ${SPACING.spacing24};
-  font-size: 2.5rem;
-  line-height: 3.25rem;
-  text-align: center;
+  padding-top: 2.125rem;
+  padding-bottom: 2.125rem;
```
Comment (jerader) [inline app/src/organisms/OnDeviceDisplay/SetupNetwork/SetWifiSsid.tsx:33]: i can't tell what these are supposed to be from the designs in figma but would be nice if they matched something in our constants. But i'm fine leaving as is!

**Review by jerader — APPROVED** [review]
lgtm! just left a question about padding.

**Comment by ewagoner** [comment]
Screenshot 2023-05-25 at 9 52 39 AM

The `InputField` element isn't totally respecting your `SSID_INPUT_FIELD_STYLE` css. For instance, it's adding an 8px padding all around, making it look comically large next to the designs. I haven't looked too in depth, but I suspect that element (which includes error handling) has baggage that the designs aren't accounting for, and I don't know if it'll be possible to override them here. I tried setting the padding to 0 manually, for example, but that didn't do what I was expecting. I suspect if you play with it just a bit longer you'll find the way.

File: app/src/organisms/OnDeviceDisplay/SetupNetwork/SetWifiSsid.tsx [hunk 2]
```diff
@@ -66,56 +84,48 @@ export function SetWifiSsid({
...
+            buttonText={i18n.format(t('shared:continue'), 'capitalize')}
+            onClick={handleContinue}
+          />
         </Flex>
       </Flex>
-
       <Flex
         flexDirection={DIRECTION_COLUMN}
         paddingX="6.34375rem"
         gridGap={SPACING.spacing12}
```
Comment (ewagoner) [inline app/src/organisms/OnDeviceDisplay/SetupNetwork/SetWifiSsid.tsx]: I think this should be shrunk to `spacing8`

**Comment by koji** [comment]
> Screenshot 2023-05-25 at 9 52 39 AM
> 
> The `InputField` element isn't totally respecting your `SSID_INPUT_FIELD_STYLE` css. For instance, it's adding an 8px padding all around, making it look comically large next to the designs. I haven't looked too in depth, but I suspect that element (which includes error handling) has baggage that the designs aren't accounting for, and I don't know if it'll be possible to override them here. I tried setting the padding to 0 manually, for example, but that didn't do what I was expecting. I suspect if you play with it just a bit longer you'll find the way.

Yeah, the issue is that the current InputField is optimized for Desktop app usage.
Now we use media query for ODD so I think we can add the style for ODD to InputField component.  
I will work on css part in a following PR.

File: app/src/organisms/OnDeviceDisplay/SetupNetwork/SetWifiSsid.tsx (hunk 1 above)
Comment (koji) [reply app/src/organisms/OnDeviceDisplay/SetupNetwork/SetWifiSsid.tsx:33]: I think I could update this part later. Right now this is using InputField without media query(for ODD).

**Review by ewagoner — APPROVED** [review]

**Pull request merged**